custom/icds_reports/sqldata/exports/take_home_ration.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import unicode_literals
+
+from sqlagg.columns import SimpleColumn
+
+from corehq.apps.reports.sqlreport import DatabaseColumn
+from corehq.apps.locations.models import SQLLocation
+from custom.icds_reports.sqldata.base import IcdsSqlData
+from custom.icds_reports.utils.mixins import ExportableMixin
+from custom.icds_reports.utils import india_now, DATA_NOT_ENTERED
+
+
+class TakeHomeRationExport(ExportableMixin, IcdsSqlData):
```

Comment:
This should use django orm not sql agg. Your call on whether you need a separate method for that (and the columns) or whether it can just be in `get_excel_data`

Reply:
Ahh. Ok. Will use ORM. Will make the columns and data in `get_excel_data`

Reply:
one question. is it because we are looking for getting rid of sql agg for dashboard tables?

Reply:
yeah, it doesn't make sense to use two different db access mechanisms unless absolutely necessary.

Reply:
ahh!! cool. 
I am done with changes. Ready for review.